linkerd/main/src/main/scala/io/buoyant/linkerd/InetSocketAddresses.scala

```diff
@@ -0,0 +1,18 @@
+package io.buoyant.linkerd
+
+import java.net.{InetAddress, InetSocketAddress, NetworkInterface}
+
+import scala.collection.JavaConverters._
+
+object InetSocketAddresses {
```

Comment:
This utility object is nice!  I might name it differently so as not to confuse it with `java.net.InetSocketAddress`.